tests/storage/storage.go

```diff
@@ -973,7 +976,15 @@ var _ = SIGDescribe("Storage", func() {
 					diskSize, err = strconv.Atoi(strings.TrimSpace(diskSizeStr))
 					diskSize = diskSize * 1000 // byte to kilobyte
 					Expect(err).ToNot(HaveOccurred())
+				})
 
+				AfterEach(func() {
+					if vmi != nil {
```

Comment:
Do we really have to wait for the VMI to be gone? Afaik it will already be in terminating state when the next test starts and it will not try to use any storage.

Reply:
Would it make sense to wait for the pod to reach the succeeded|failed state instead?

Reply:
hm... maybe... but I though it was the same as waiting for the VMI to dissapear. Can it happen that virt-launcher pod will be still running while VMI has alreday gone?

Reply:
Yes, but I meant the pod with the preStop hook. Since that is the one which will do the unmount and which we want to wait for.

Reply:
ah... this is done a couple of lines bellow:
```go
tests.WaitForPodToDisappearWithTimeout(pod.Name, 120)
```
So at first the VMI is deleted (with wait) then the hostpath pod is deleted (also with wait: that triggers the preSstop hook and calls umount).

Reply:
Ah ok thanks.